src/wp-includes/block-template-utils.php

```diff
@@ -810,27 +810,27 @@ function get_block_template( $id, $template_type = 'wp_template' ) {
 }
 
 /**
- * Retrieves a single unified template object using its id.
+ * Retrieves an array of template objects by id.
```

Comment:
Running this outputs a single WP_Block_Template object:

```php
get_block_file_template( 'twentytwentytwo//page' )
```
@johnbillion Can you clarify "an array of template objects by id" for me?

Reply:
This is indeed not right. The types throughout the docs are incorrect.